samples/tab-external-auth/csharp/TabExternalAuth/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,124 @@
+        }
+
+        .card {
+            box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+            width: 30rem;
+            height: 12rem;
+            margin-left: 2rem;
+            margin-top: 0.5rem;
+        }
+
+            .card:hover {
+                box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+            }
+
+        .container {
+            padding: 2px 2px;
+            display: flex;
+        }
+
+        img {
+            border-radius: 50%;
+        }
+
+        .profile {
+            margin-top: 1rem;
+            margin-left: 2rem;
+            max-width: 30rem;
+            align-content: flex-start;
+        }
+
+        .btn-outline-info {
+            margin-left: 19rem;
+            max-width: 10rem;
+        }
+
+        .signin-header {
+            margin-left: 2rem;
+            margin-top: 1rem;
+        }
+
+        .card-container-div {
+            margin-left: 30%;
+            padding-bottom: 2rem;
+        }
+
+        .card-container-div-mobile {
+            margin-left: 1%;
+            padding-bottom: 2rem;
+        }
+
+        #divError {
+            margin-left: 1rem
+        }
+    </style>
+    <script>
+        // Google app client id
+        var googleId = "@ViewBag.GoogleAppId"; // Google app client id
+
+        // Redirect url path
+        var url = window.location.origin + "/Auth/GoogleEnd"; 
+
+        // Fetch id token
+        function getGoogleIdToken() {
+            return new Promise((resolve, reject) => {
+                microsoftTeams.app.initialize().then(() => {
+                    microsoftTeams.authentication.authenticate({
+                        url: `https://accounts.google.com/o/oauth2/v2/auth?redirect_uri=${url}&state={"authId": ${googleId},"oauthRedirectMethod":"deeplink"}&client_id=${googleId}&scope=openid email profile&response_type=code`,
+                        width: 600,
+                        height: 535
+                    }).then((result) => {
+                        let data = localStorage.getItem(result);
+                        let tokenDetails = JSON.parse(data);
+                        localStorage.removeItem(result);
+                        resolve(tokenDetails);
+                    }).catch((reason) => {
+                        console.log("failed" + reason);
+                        reject(reason);
+                    })
+                })
+            })
+        }
+    </script>
+</head>
+
+<body class="theme-light">
+    <div class="surface">
+        <h3> Welcome to Tab External Auth Sample</h3>
+        <div class="card-container-div">
+            <div>
+                <div class="signin-header"><b>Sign in to Google</b></div>
+                <div class="card">
+                    <div class="container">
+                        <img src="https://media.istockphoto.com/vectors/profile-placeholder-image-gray-silhouette-no-photo-vector-id1016744034?k=20&m=1016744034&s=612x612&w=0&h=kjCAwH5GOC3n3YRTHBaLDsLIuF8P3kkAJc9RvfiYWBY="
```

Comment:
What is this image ?

Reply:
Its a placeholder image for default page